File: Android/RecyclerViewDay/app/src/main/java/com/example/recyclerviewday/MainActivity.java
package com.example.recyclerviewday;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    ArrayList<Product> products;
    RecyclerView recyclerProducts;
    ProductAdapter productAdapter;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        initview();
        initPrducts();


    }

    private void initview() {
        recyclerProducts=findViewById(R.id.recyclerProduct);
        recyclerProducts.setLayoutManager(new LinearLayoutManager(
                this,
                LinearLayoutManager.VERTICAL,
                false
        ));
       // productAdapter =new ProductAdapter()(products);
       // recyclerProducts.setAdapter(productAdapter);
    }

    private void initPrducts() {

    }
}

File: Android/UiAppStore/app/src/main/java/com/uiappstore/onboarding/ui/activities/MainActivity.kt
package com.uiappstore.onboarding.ui.activities

import android.net.Uri
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.provider.ContactsContract.CommonDataKinds.Website.URL
import android.widget.MediaController
import androidx.databinding.DataBindingUtil
import com.uiappstore.R
import com.uiappstore.databinding.ActivityMainBinding
import kotlinx.coroutines.CoroutineScope
import kotlinx.coroutines.Dispatchers
import kotlinx.coroutines.coroutineScope
import kotlinx.coroutines.launch

class MainActivity : AppCompatActivity(){

    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding=DataBindingUtil.setContentView(this,R.layout.activity_main)
        CoroutineScope(Dispatchers.IO).launch {
            initVideoView()        }

    }
    private fun initVideoView()
    {
        val path:String="Android.Resources"
       val uri=Uri.parse(path)
        binding.video.setVideoURI(uri)
        val mediaController=MediaController(this)
        binding.video.setMediaController(mediaController)
    }

}